source/docs/advanced/production-ready/metrics.md

```diff
@@ -11,9 +11,37 @@ Front-Commerce provides a `/metrics` endpoint with data about the node process a
 
 You need two steps to enable this endpoint
 
-- `npm install express-prom-bundle prom-client`
+- `npm install express-prom-bundle prom-client --save`
```

Comment:
Why the `--save` option? since version 5 this is automatic https://blog.npmjs.org/post/161081169345/v500
> [npm will --save by default now](https://twitter.com/maybekatz/status/859229741676625920). Additionally, package-lock.json will be automatically created unless an npm-shrinkwrap.json exists. ([#15666](https://github.com/npm/npm/pull/15666))

and v5 was last seen at node@8.11.4

Reply:
My mind does not wish to admit it -_- 
removed :+1: